graphql/input.go

```diff
@@ -8,15 +8,15 @@ import (
 
 const unmarshalInputCtx key = "unmarshal_input_context"
 
-func BuildMap(unmarshaler ...interface{}) map[reflect.Type]reflect.Value {
+// BuildUnmarshalerMap returns a map of unmarshal functions of the ExecutableContext
+// to use with the WithUnmarshalerMap function.
+func BuildUnmarshalerMap(unmarshaler ...interface{}) map[reflect.Type]reflect.Value {
 	maps := make(map[reflect.Type]reflect.Value)
 	for _, v := range unmarshaler {
 		ft := reflect.TypeOf(v)
-		if ft.Kind() != reflect.Func {
-			panic("unmarshaler must be a function")
+		if ft.Kind() == reflect.Func {
```

Comment:
I prefer this avoiding of panics, but if the kind is not a function... what now happens when nothing is added to the maps here?